src/globals/spacing-option.ts

```diff
@@ -0,0 +1 @@
+export type spacingOption = 1 | 2 | 3 | 4 | 5 | 6;
```

Comment:
I think a docblock would be useful here. Does this look good to you @tomwpaterson-shape ?

```suggestion
/**
 * These are option identifiers, not exact measurements. 
 * Exact measurements are currently set in variables.scss
 */
export type spacingOption = 1 | 2 | 3 | 4 | 5 | 6;
```

Also should we add 0 to the list of possible options? variables.scss defines it as a zero length space:
https://github.com/thebiggive/components/blob/57815b9d6262d3e4fc85a3820876bc1556acb3ac/src/globals/variables.scss#L52